### ISSUE TITLE
Godoc for lints package malformed

### ISSUE THREAD

**Issue by jsha** [zmap/zlint#210]
On https://godoc.org/github.com/zmap/zlint/lints, it looks like some of the comments are being turned into godoc in unexpected ways:

```
lint_rsa_no_public_key.go

lint_sub_cert_certificate_policies_marked_critical.go ***************************************************************************** BRs: 7.1.2.3 certificatePolicies This extension MUST be present and SHOULD NOT be marked critical. *****************************************************************************
```

**Comment by justinbastress** [comment]
I think the real issue is that zlint just doesn't do godocs -- but for the short term, we can probably clean up that `lints` page by moving the copyright / BR citations down below the `package` lines.

**Comment by zakird** [comment]
Looks like this was addressed by #213.
